### PR TITLE
chore: added new FeatureFlags

### PR DESCRIPTION
**Pull request by akshbhu**
*Description of changes:*

* Added new Feature flags 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: client/src/amplify-ui/cli-feature-flag/feature-flags.json [hunk 1]
```diff
@@ -42,6 +42,126 @@
             "defaultExistingProject": false
           }
         ]
+      },
+      "useExperimentalPipelinedTransformer": {
+        "description": "Use pipeline feature in new Transformer.",
```
Comment (renebrandel) [inline client/src/amplify-ui/cli-feature-flag/feature-flags.json:47]:
```suggestion
        "description": "Use pipeline resolvers in the new GraphQL Transformer.",
```

File: client/src/amplify-ui/cli-feature-flag/feature-flags.json [hunk 2]
```diff
@@ -42,6 +42,126 @@
...
+        "type": "Feature",
+        "valueType": "Boolean",
+        "versionAdded": "4.32.4",
+        "deprecationDate": "May 1st 2021",
+        "values": [
+          {
+            "value": "true",
+            "description": "Allows to use pipeline feature in transfomer.",
```
Comment (renebrandel) [inline client/src/amplify-ui/cli-feature-flag/feature-flags.json:55]:
```suggestion
            "description": "Enable pipeline resolvers.",
```

File: client/src/amplify-ui/cli-feature-flag/feature-flags.json [hunk 3]
```diff
@@ -42,6 +42,126 @@
...
+            "defaultNewProject": true,
+            "defaultExistingProject": false
+          },
+          {
+            "value": "false",
+            "description": "previous graphql transformer.",
```
Comment (renebrandel) [inline client/src/amplify-ui/cli-feature-flag/feature-flags.json:61]:
```suggestion
            "description": "Disables pipeline resolver-based GraphQL Transformer.",
```

File: client/src/amplify-ui/cli-feature-flag/feature-flags.json [hunk 4]
```diff
@@ -42,6 +42,126 @@
...
+            "defaultNewProject": false,
+            "defaultExistingProject": true
+          }
+        ]
+      },
+      "enableIterativeGsiUpdates": {
+        "description": "Enable updating more than 1 GSIs in update stage .",
```
Comment (renebrandel) [inline client/src/amplify-ui/cli-feature-flag/feature-flags.json:68]:
```suggestion
        "description": "Enable multiple GSI updates through iterative incremental deployments.",
```

File: client/src/amplify-ui/cli-feature-flag/feature-flags.json [hunk 5]
```diff
@@ -42,6 +42,126 @@
...
+        "type": "Feature",
+        "valueType": "Boolean",
+        "versionAdded": "4.38.0",
+        "deprecationDate": "May 1st 2021",
+        "values": [
+          {
+            "value": "true",
+            "description": "Projects can mutate more than 1 GSIs in update flow.",
```
Comment (renebrandel) [inline client/src/amplify-ui/cli-feature-flag/feature-flags.json:76]:
```suggestion
            "description": "GraphQL schemas changes can contain multiple GSI changes.",
```

File: client/src/amplify-ui/cli-feature-flag/feature-flags.json [hunk 6]
```diff
@@ -42,6 +42,126 @@
...
+            "description": "Throws Error when mutating more than 1 GSIs in update flow. .",
+            "defaultNewProject": false,
+            "defaultExistingProject": true
+          }
+        ]
+      }
+    }
+  },
+  "frontend-ios": {
+    "description": "Feature Flag related to Frontend Ios",
```
Comment (renebrandel) [inline client/src/amplify-ui/cli-feature-flag/feature-flags.json:91]:
```suggestion
    "description": "Feature Flag related to iOS projects",
```

File: client/src/amplify-ui/cli-feature-flag/feature-flags.json [hunk 7]
```diff
@@ -42,6 +42,126 @@
...
+    "features": {
+      "enableXcodeIntegration": {
+        "description": "Enable xCode integration .",
```
Comment (renebrandel) [inline client/src/amplify-ui/cli-feature-flag/feature-flags.json:94]:
```suggestion
        "description": "Automatically add DataStore / API models to the Xcode project.",
```

File: client/src/amplify-ui/cli-feature-flag/feature-flags.json [hunk 8]
```diff
@@ -42,6 +42,126 @@
...
+        "type": "Feature",
+        "valueType": "Boolean",
+        "versionAdded": "4.40.0",
+        "deprecationDate": "May 1st 2021",
+        "values": [
+          {
+            "value": "true",
+            "description": "Enables Xcode integration.",
```
Comment (renebrandel) [inline client/src/amplify-ui/cli-feature-flag/feature-flags.json:102]: All newly generated DataStore/API models are auto-added to the Xcode project.

File: client/src/amplify-ui/cli-feature-flag/feature-flags.json [hunk 9]
```diff
@@ -42,6 +42,126 @@
...
+            "defaultNewProject": true,
+            "defaultExistingProject": false
+          },
+          {
+            "value": "false",
+            "description": "Doesnt allows Xcode integration",
```
Comment (renebrandel) [inline client/src/amplify-ui/cli-feature-flag/feature-flags.json:108]:
```suggestion
            "description": "All newly generated DataStore/API models are NOT auto-added to the Xcode project.",
```

File: client/src/amplify-ui/cli-feature-flag/feature-flags.json [hunk 10]
```diff
@@ -42,6 +42,126 @@
...
+          }
+        ]
+      }
+    }
+  },
+  "auth": {
+    "description": "Feature Flag related to auth",
+    "features": {
+      "enableCaseInsensitivity": {
+        "description": "enable case insensitivity .",
```
Comment (renebrandel) [inline client/src/amplify-ui/cli-feature-flag/feature-flags.json:120]:
```suggestion
        "description": "Toggles case insensitivity for Cognito login mechanisms (username, email etc.)",
```

File: client/src/amplify-ui/cli-feature-flag/feature-flags.json [hunk 11]
```diff
@@ -42,6 +42,126 @@
...
+        "type": "Feature",
+        "valueType": "Boolean",
+        "versionAdded": "4.40.0",
+        "deprecationDate": "May 1st 2021",
+        "values": [
+          {
+            "value": "true",
+            "description": "enables case insensitivity for auth",
```
Comment (renebrandel) [inline client/src/amplify-ui/cli-feature-flag/feature-flags.json:128]:
```suggestion
            "description": "Emails and usernames are case insensitive",
```

File: client/src/amplify-ui/cli-feature-flag/feature-flags.json [hunk 12]
```diff
@@ -42,6 +42,126 @@
...
+            "defaultNewProject": true,
+            "defaultExistingProject": false
+          },
+          {
+            "value": "false",
+            "description": "Disables case insensitivity for auth.",
```
Comment (renebrandel) [inline client/src/amplify-ui/cli-feature-flag/feature-flags.json:134]:
```suggestion
            "description": "Emails and usernames are case sensitive.",
```

File: client/src/amplify-ui/cli-feature-flag/feature-flags.json (hunk 1 above)
Comment (akshbhu) [reply client/src/amplify-ui/cli-feature-flag/feature-flags.json]: done!

File: client/src/amplify-ui/cli-feature-flag/feature-flags.json (hunk 4 above)
Comment (akshbhu) [reply client/src/amplify-ui/cli-feature-flag/feature-flags.json]: done!

File: client/src/amplify-ui/cli-feature-flag/feature-flags.json (hunk 5 above)
Comment (akshbhu) [reply client/src/amplify-ui/cli-feature-flag/feature-flags.json]: done!

File: client/src/amplify-ui/cli-feature-flag/feature-flags.json (hunk 6 above)
Comment (akshbhu) [reply client/src/amplify-ui/cli-feature-flag/feature-flags.json]: done!

File: client/src/amplify-ui/cli-feature-flag/feature-flags.json (hunk 7 above)
Comment (akshbhu) [reply client/src/amplify-ui/cli-feature-flag/feature-flags.json]: done!

File: client/src/amplify-ui/cli-feature-flag/feature-flags.json (hunk 8 above)
Comment (akshbhu) [reply client/src/amplify-ui/cli-feature-flag/feature-flags.json]: done!

File: client/src/amplify-ui/cli-feature-flag/feature-flags.json (hunk 9 above)
Comment (akshbhu) [reply client/src/amplify-ui/cli-feature-flag/feature-flags.json]: done!

File: client/src/amplify-ui/cli-feature-flag/feature-flags.json (hunk 10 above)
Comment (akshbhu) [reply client/src/amplify-ui/cli-feature-flag/feature-flags.json]: done!

File: client/src/amplify-ui/cli-feature-flag/feature-flags.json (hunk 11 above)
Comment (akshbhu) [reply client/src/amplify-ui/cli-feature-flag/feature-flags.json]: done!

File: client/src/amplify-ui/cli-feature-flag/feature-flags.json (hunk 12 above)
Comment (akshbhu) [reply client/src/amplify-ui/cli-feature-flag/feature-flags.json]: done!

**Review by renebrandel — APPROVED** [review]

**Review by kaustavghosh06 — APPROVED** [review]
lgtm.

**Pull request merged**